app/models/vendor_api/rejection_reasons.rb

```diff
@@ -17,6 +17,10 @@ def initialize(reasons_attrs = [])
         reason
       end
     end
+
+    def self.reference_data
+      CODES.map { |code, hash| { code: code, label: hash[:label], default_details: hash.dig(:details, :text) } }
```

Comment:
Did you mean:
```suggestion
      CODES.map { |code, hash| { code: code, label: hash[:label], default_details: hash.dig(:details, :label) } }
```
:text is no longer in the yml file so I'm getting null on the response. Also, as per your screenshot, is the yml file supposed to have longer descriptions of the rejection code?

Reply:
Yeah sorry this needs rebase since https://github.com/DFE-Digital/apply-for-teacher-training/commit/9202a3aa94ab6dd1cda83410c7686cd04148833c

Reply:
@JonF1 rebased, this PR should make a bit more sense with the default details text in the config now.